### ISSUE TITLE
[Accessibility]A11y_ AzureDataStudio_ NewNotebook_Keyboard: Keyboard focus is not visible after "Clear Results" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5611]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details**:
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool name: Keyboard
MAS Number: MAS2.4.7

**Steps to Reproduce:**
1. Install the app Azure Data Studio
2. Connect to the Database->Go to file->New Notebook
3. Navigate to the "Clear Results" button through tab.
4. Observe that after "Clear Results" button ,keyboard focus is not visible.

**Actual:**
Navigate till  the "Clear Results" button through tab ,after clicking  tab again the keyboard focus is not visible .When tab press again it is visible on “Code ” link .

**Expected:**
After clicking on tab the focus should move to the “Code ” link.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Keyboard only users find it difficult to navigate the page.

**MAS Reference:**
**MAS2.4.7 :**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment for Reference:**
[5611_[Accessibility]A11y_ Azure Data Studio_ NewNotebook_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3215182/5611_.Accessibility.A11y_.Azure.Data.Studio_.NewNotebook_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
Dupe of #5616.

**Comment by AccessibilityTestingTeam-TCS** [comment]
This issue is observed in "Windows". The issue is related to the loss of focus. When the keyboard focus is on "Clear Results" button then after clicking on tab key the focus is not visible on +Code button.
So, this is not the expected behaviour, the focus should be visible on each actionable control without any loss.
